### ISSUE TITLE
[Feat. Request] Task States

### ISSUE THREAD

**Issue by melmaninga** [roovo/obsidian-card-board#122]
Would love it if it could handle multiple states.  With my old-school analog way of task management, I use symbols to give me a feeling of traction.  I realise this can be handled by tags, with a css snippet it would make my Cardboard so much more visual and a perfect replica of my analog method.

- [ ] Unchecked
- [+] In Progress
- [?] Need More Info
- [>] Scheduled / Deferred / Delegated
- [!] Important
- [x] Completed

**Comment by roovo** [comment]
I thought I already had an issue for this, but I don't seem to (there is a discussion tho #66) sop thank you for adding.  I would defo like to support this - as I see that some themes support this concept too.

After I've finished off my current few items I will be working on drag and drop, so it will be a while before I get to this but I am tagging it as accepted and needing some thought (as I will need to ensure that I will be compatible with themes).

Many thanks.  I'll keep this issue updated as there is any progress or with any related thoughts I have before I get to working on it.

**Comment by roovo** [comment]
as mentioned in #130 - some support for this is in hand in the Tasks plugin:

- [Test build of Tasks with custom status support - for testing and feedback](https://github.com/obsidian-tasks-group/obsidian-tasks/discussions/1530)
- [Project: Custom Statuses for Tasks Plugin](https://github.com/orgs/obsidian-tasks-group/projects/2/views/2)

**Comment by roovo** [comment]
From discord:

ClareMacrae
 — 
Yesterday at 23:03
I'm aware of the following themes and snippets for custom checkboxes in Obsidian:
Minimal Theme:
https://minimal.guide/Block+types/Checklists

ITS Theme:
https://github.com/SlRvb/Obsidian--ITS-Theme/tree/main

SlRvb's Alternate Checkboxes snippet - same options as ITS Theme...
https://github.com/SlRvb/Obsidian--ITS-Theme/blob/main/Guide/Alternate-Checkboxes.md

Are there any other commonly-used ones I should know about?

ypsyav
 — 
Today at 00:00
the AnuPpuccin theme has alternate checkboxes.

ClareMacrae
 — 
Today at 00:03
Ah interesting. Thank you. That references the Primary theme, whose custom checkboxes I was also unaware of. 
https://github.com/ceciliamay/obsidianmd-theme-primary#-alternative-checkboxes 
This is really useful @gypsyav - thank you!
Oddly Primary does not show up as available on my Phone.
efemkay
 — 
Today at 00:53
i believe if you don't already have it installed, primary theme won't show up in community theme search. it's still in "legacy" mode, not yet updated for obsidian v1
https://github.com/AnubisNekhet/AnuPpuccin

**Comment by ltroj** [comment]
It would be great if CardBoard supported custom statuses because atm adding a custom status to a task removes it from the Boards since the task is not recognized as such anymore.

**Comment by roovo** [comment]
Yeah - I may well bump this up the stack as this is prob happening for a few people now that Tasks supports these.  I would prob prioritize reading them so they appear on the boards with theme custom checkboxes and leave being able to change the state to a later release - as I don't want to minimize the other work I take on whilst I get the drag and drop stuff out of the way.

**Comment by Bjaux** [comment]
@roovo also it seems to me that changing the state would maybe greatly build upon the drag'n drop stuff. If that is done we could have a "new" board type by custom checkbox status, where dragging and droping would simply change the checkbox :)

**Comment by roovo** [comment]
see  also: #66
